src/workers/postUpdated.ts

```diff
@@ -0,0 +1,68 @@
+import { messageToJson, Worker } from './worker';
+import { ArticlePost, SharePost } from '../entity';
+
+interface Data extends ArticlePost {
+  updated_at: Date;
+}
+
+const worker: Worker = {
+  subscription: 'api.post-updated',
+  handler: async (message, con, logger): Promise<void> => {
+    const data: Data = messageToJson(message);
+    try {
+      const { id, updated_at } = data;
+      const updatedDate = new Date(updated_at);
+      await con.transaction(async (entityManager) => {
+        const databasePost = await entityManager
+          .getRepository(ArticlePost)
+          .findOneBy({ id });
+
+        if (
+          !databasePost ||
+          databasePost.metadataChangedAt.toISOString() >=
+            updatedDate.toISOString()
+        ) {
+          return;
+        }
+
+        const becomesVisible = !databasePost?.visible && !!data?.title?.length;
+
+        await entityManager.getRepository(ArticlePost).update(
+          { id: databasePost.id },
+          {
+            summary: data?.summary,
+            description: data?.description,
+            readTime: data?.readTime,
+            toc: data?.toc,
+            keywords: data?.keywords,
+            siteTwitter: data?.siteTwitter,
+            creatorTwitter: data?.creatorTwitter,
+            sourceId: data?.sourceId,
+            image: data?.image,
+            title: data?.title,
+            metadataChangedAt: updatedDate,
```

Comment:
One thing that I noticed is that we change this value ourselves in this CDC in some cases. 
I think we should remove some of this cases from CDC (for example when `visible` changes) because it addresses here and it may create race conditions.

Reply:
We can remove the other ones if that's what you mean.
Since those are not actually using it.

Reply:
Tinybird exporter uses it. I'll review it now again and let you know what props can be ignored in the cdc

Reply:
I believe we can keep only this lines:
isChanged(data.payload.before, data.payload.after, 'deleted') ||
      isChanged(data.payload.before, data.payload.after, 'banned') ||
      isChanged(data.payload.before, data.payload.after, 'tagsStr')